configurator.js

```diff
@@ -125,9 +125,9 @@ Configurator.prototype._defaultConfiguration = function () {
 }
 
 Configurator.prototype._pickFirst = function (propName) {
-  var values = [].slice.call(arguments, 1),
-    i = 0,
-    value
+  var values = [].slice.call(arguments, 1)
+  var i = 0
+  var value
```

Comment:
I'm all for being explicit. I updated the PR to initialize it to `null` to indicate the absence of a value, and is also false-y.